pkg/cli/secretsencrypt/secrets_encrypt.go

```diff
@@ -214,3 +214,25 @@ func Reencrypt(app *cli.Context) error {
 	fmt.Println("reencryption started")
 	return nil
 }
+
+func RotateKeys(app *cli.Context) error {
+	var err error
```

Comment:
I'm not sure if this declaration is necessary.

Reply:
Thanks!